src/components/Book.js

```diff
@@ -3,16 +3,41 @@ import React from 'react';
 const Book = (props) => {
     const sessions = props.book.sessions
     const timestamp = sessions[sessions.length - 1].timestamp
-    // console.log(timestamp)
     const dateCompleted = new Date(timestamp)
-    // console.log(dateCompleted)
+
+    // go through the sessions array for each book and find the total time spent reading the book
+
+    // props.book.sessions.reduce((previousTotal, object) => previousTotal + object.desired_value, initialValue)
```

Comment:
I would suggest that you remove this comment, which is a "how do I do it" comment, because you have already done it. You could write the comment to match the content and style of the other comments below. Or see alternative above.